### ISSUE TITLE
Prepare Git/GitHub training

### ISSUE THREAD

**Issue by ccarouge** [ACCESS-NRI/Land-workshop-prep#5]
We need training that is tailored for the use of Git and GitHub for CABLE.

In the CABLE documentation, there is already a contribution flowchart that presents a workflow to contribute: https://cable.readthedocs.io/en/latest/developer_guide/contribution_flowchart/

It relies on people setting up Git/GitHub according to these instructions: https://access-nri.github.io/Training/HowTos/GitAndGitHub/ In particular, it requires people to set up auto-remote for git.

**Comment by bschroeter** [comment]
Do we want to come up with a trivial example of checking out the code, making some changes, then kicking off the review process in order to get it merged back? Perhaps with and without a merge conflict and how to resolve it?

**Comment by ccarouge** [comment]
Yes, I think that would be great. Can you try to figure something out using a fork of https://github.com/CABLE-LSM/CABLE?

**Comment by ccarouge** [comment]
Or do you think it is worth looking for an example that makes some sense?

**Comment by bschroeter** [comment]
I'm working on putting this into the ACCESS-NRI ppt template. I'll work back through the training I did for BoM some years ago and see what is relevant.
